crates/ruma-common/tests/events/audio.rs

```diff
@@ -272,20 +265,12 @@ fn plain_content_deserialization() {
         }
     });
 
-    assert_matches!(
-        from_json_value::<AudioEventContent>(json_data)
-            .unwrap(),
-        AudioEventContent {
-            message,
-            file,
-            audio: AudioContent { duration: None, waveform: Some(waveform), .. },
-            ..
-        }
-        if message.find_plain() == Some("Upload: my_new_song.webm")
-            && message.find_html().is_none()
-            && file.url == "mxc://notareal.hs/abcdef"
-            && waveform.amplitudes().len() == 52
-    );
+    let content = from_json_value::<AudioEventContent>(json_data).unwrap();
+    assert_eq!(content.message.find_plain(), Some("Upload: my_new_song.webm"));
+    assert!(content.message.find_html().is_none());
```

Comment:
This should probably use `assert_eq` or `assert_matches` as well, so the find_html result is printed if it's `Some(_)`.